time timestamp
float32 xl_x
float32 xl_y
float32 xl_z
float32 mag_x
float32 mag_y
float32 mag_z
